### PR TITLE
ahkuma/poidwed

### PR DESCRIPTION
**Pull request by ahkuma**

### CODE REVIEW

File: src/javascript/app/pages/user/account/authenticate.js [hunk 1]
```diff
@@ -39,6 +39,11 @@ const Authenticate = (() => {
                         link = Url.urlForCurrentDomain('https://marketing.binary.com/authentication/MF_Authentication_Process.pdf');
                     }
                     $not_authenticated.find('.learn_more').setVisibility(1).find('a').attr('href', link);
+
+                    if (isIdentificationNoExpiry(Client.get('residence'))) {
+                        $('#expiry_datepicker_proofid').setVisibility(0);
+                        $('#exp_date_2').datepicker('setDate', '2099-12-31');
```
Comment (negarn) [inline src/javascript/app/pages/user/account/authenticate.js]: does this work for mobile too? or just desktop?

Comment (ahkuma) [reply src/javascript/app/pages/user/account/authenticate.js]: @negarn It should. Seems like it use the same jQuery date picker for both.

Comment (negarn) [reply src/javascript/app/pages/user/account/authenticate.js]: @ahkuma please test it. in mobile we use the native datepicker.

Comment (ahkuma) [reply src/javascript/app/pages/user/account/authenticate.js]: Sorry, I should have said, I tested it before and it works fine. I tested it again using both the mac and my phone and it works fine.

**Pull request merged**